jsmith/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="scripts/jquery-3.2.1.min.js"></script>
+  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <link href="css/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Bungee+Inline" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Shrikhand" rel="stylesheet">
+  <title> Home </title>
+</head>
+
+<body>
+
+  <div class="nav">
+    <ul>
+      <li class="nav-link"><a href="index.html">Home</a></li>
+      <li class="nav-link"><a href="artwork.html">Artwork</a></li>
+      <li class="nav-link"><a href="http://saskartists.ca/">Sask Artists</a></li>
+    </ul>
+  </div>
+
+  <div class="container">
+    <p class="portrait"> Portrait</p>
+    <a>  <img src="images/doggrass.jpg" class="picture" id="frontpageimg" alt="A photo of John Shhit Smith"/></a>
+    <h1>  John Shhit Smith </h1>
+    <i class="fa fa-chevron-down fa-2x" id="arrowdown"></i>
+  </div>
+
+  <div class="lower">
+    <h1 class="underline"> Bio </h1><br/><br/>
+    <p> John Shhit Smith is supposedly from Regina. </p>
```

Comment:
I recognize that this artist wanted to remain anonymous; however, this does not mean that we know nothing about them. While we do not know any personal details, we do know about what kind of art they make, as we have examples of it on our site. You may want to consider supplementing this section with that information.